fr/common.json

```
@@ -0,0 +1,452 @@
+      "description": "Les taux de victoire et les pourcentages de classement moyen ci-dessous sont généralement plus élevés que les mesures globales\npuisque les exemples ici n'incluent pas les correspondances pour les personnages en dessous du niveau 10.",
+      "averageDPM": "DPM Moyen"
+    }
+  },
+  "itemSearchBox": {
+    "anySubcategory": "Toutes les catégories",
+    "anyItemOptions": "Toutes les options",
+    "anyItemSkills": "Toutes les compétences d'objet",
+    "searchPlaceholder": "Chercher..."
+  },
+  "players": {
+    "title": "{{name}} - Joueur",
+    "playerNotFound": "Joueur  {{name}} introuvable",
+    "matchEmpty": "Aucune partie trouvée !",
+    "connectionFailed": "Échec de connexion.",
+    "recentUpdateDate": "À jour",
+    "requestUpdate": "Mettre à jour",
+    "inMaintaining": "La mise à jour est en maintenance...",
+    "noPlayRecordShort": "Aucune donnée",
+    "openWidgetModal": "Widget Stream",
+    "widgetModal": {
+      "title": "Widget Stream",
+      "description": "Il s'agit d'un widget affichant votre MMR actuelle pour votre stream. Utilisez-le avec un logiciel de streaming comme OBS ou XSplit.",
+      "windowSizeDescription": "Taille minimum suggérée : 240 x 140(px)",
+      "options": "Options",
+      "informationType": "Information",
+      "informationTypeOptions": {
+        "account": "Rang",
+        "accountAndRecent3Matches": "Rang + 3 dernières parties"
+      },
+      "backgroundType": "Arrière plan",
+      "backgroundTypeOptions": {
+        "tier": "Rang",
+        "halfTransparent": "Noir Semi-Transparent"
+      },
+      "preview": "Prévisualisation",
+      "copyURL": "Copier"
+    },
+    "seasonTab": {
+      "every": "Tout",
+      "rankSeason": "Saison {{seasonNumber}}",
+      "rankPreseason": "Présaison {{seasonNumber}}",
+      "normal": "Normal"
+    },
+    "hourPostfix": "{{hour}} heures",
+    "cannotCalculate": "NC",
+    "rankInPlacement": "En placement",
+    "rankPercentage": "TOP {{percentage}}%",
+    "rankPercentagePrefix": "TOP",
+    "notEnoughSamples": "NC",
+    "totalWins": "Victoires",
+    "rank": "{{rank, ordinal}}",
+    "totalDeal": "Dégâts",
+    "detail": {
+      "notEnoughSamplesToCalculate": "Pas assez d'échantillons à calculer.",
+      "noPlayerData": "Aucune information. Veuillez chercher ce joueur.",
+      "averageMMRDifferencePrefix": "Ma MMR",
+      "averageMMR": "MMR Moyenne",
+      "averageExpGain": "Gain d'EXP moyen",
+      "match": "Données générales",
+      "rank": "Classement",
+      "name": "Joueur",
+      "mmr": "MMR",
+      "routeId": "ID de L'itinéraire",
+      "noRouteIdOfStart": "Itinéraire Privé",
+      "totalDeal": "Dégâts",
+      "equipments": "Équipement",
+      "matchVersion": "Version",
+      "matchId": "ID du Match",
+      "playerKilledMe": "M'a tué",
+      "iKilledPlayer": "Tué par moi",
+      "playerKilledTeammate": "Tué par un allié",
```

This field is for those who killed one's ally.

```suggestion
      "playerKilledTeammate": "Tué un allié",
```

---

The error got corrected